php-practice/section10.php

```diff
@@ -76,6 +89,16 @@
  * associative array with `name` and `genre` indexes) and loops over it, printing out the name of each other.
  */
 
+function authorName(array $favoriteAuthor) {
+      foreach ($favoriteAuthor as $x => $x_value) {
+      echo $x_value['name'];
+      echo PHP_EOL;
+    }
+}
```

Comment:
Good! This is correct code.

Here are some changes I would have made to increase the clarity of the code:

1. Change the name of the parameter / variable to `$favoriteAuthors` (plural)
2. Change the name of the function to something like `printAuthorNames`. Function names should contain a verb and should make it stupidly obvious what they do. Part of the reason for this is that often in a real program when someone calls a function it is defined in a completely different file. So if the function name is confusing or unclear, you have to go look up the function and **read most or all of its code** just to understand what it's doing.